### PR TITLE
Release/1.20.3

### PR DESCRIPTION
**Pull request by ThomasSession**
Release code for v1.20.3

### CODE REVIEW

File: app/src/main/java/org/thoughtcrime/securesms/components/menu/ActionItem.kt [hunk 1]
```diff
@@ -4,6 +4,7 @@ import android.content.Context
 import androidx.annotation.AttrRes
 import androidx.annotation.ColorInt
 
+
```
Comment (AL-Session) [inline app/src/main/java/org/thoughtcrime/securesms/components/menu/ActionItem.kt:7]: Can remove, just an errant enter..

File: app/build.gradle [hunk 2]
```diff
@@ -268,9 +281,9 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-viewmodel-compose:$lifecycleVersion"
     implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
     implementation "androidx.paging:paging-runtime-ktx:$pagingVersion"
-    implementation 'androidx.activity:activity-ktx:1.5.1'
-    implementation 'androidx.activity:activity-compose:1.5.1'
-    implementation 'androidx.fragment:fragment-ktx:1.5.3'
+    implementation 'androidx.activity:activity-ktx:1.9.2'
```
Comment (AL-Session) [inline app/build.gradle:284]: Is there an updated BOM we can use rather than specifying versions individually?

**Review by SessionHero01 — APPROVED** [review]
Congrats on the very first PR!

File: app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt [hunk 3]
```diff
@@ -1528,14 +1537,11 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
         sendEmojiRemoval(emoji, message)
     }
 
+    /**
```
Comment (AL-Session) [inline app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt:1540]: Could just be a single line comment via `//`

File: app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt [hunk 4]
```diff
@@ -2066,88 +2072,14 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
...
-    }
-
-    private fun showDeleteLocallyUI(messages: Set<MessageRecord>) {
-        showSessionDialog {
-            title(resources.getQuantityString(R.plurals.deleteMessage, messages.count(), messages.count()))
-            text(resources.getString(R.string.deleteMessagesDescriptionDevice))
-            button(R.string.delete) { messages.forEach(viewModel::deleteLocally); endActionMode() }
-            cancelButton(::endActionMode)
-        }
+        selectMessage(messages.first(), 0) //TODO: begin selection mode
```
Comment (AL-Session) [inline app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt:2075]: TODO still req'd?

**Review by AL-Session — APPROVED** [review]
Nothing stopping merge - lessgo.

File: app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt (hunk 3 above)
Comment (ThomasSession) [reply app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt:1540]: This is the format to declare the description of the method. This way it shows up on docs or when looking up methods

File: app/build.gradle (hunk 2 above)
Comment (ThomasSession) [reply app/build.gradle:284]: As far as I know the BOM is only for Compose, I don't think it includes androidx activity

File: app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt (hunk 4 above)
Comment (ThomasSession) [reply app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt:2075]: Good question... This is a remnant from the old code, though I'm not sure what they meant by it.
I'll remove the comment

File: app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt (hunk 3 above)
Comment (AL-Session) [reply app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt:1540]: Okey-doke. Maybe we should be using that throughout then. Happy to leave it for now.

File: app/build.gradle (hunk 2 above)
Comment (AL-Session) [reply app/build.gradle:284]: Fair point - I thought we could grab a "Bill-of-Materials" for other, non-compose stuff so we could be confident it all works nicely together - but nope! Resolving.

**Pull request merged**